### ISSUE TITLE
App crashing without Internet 

### ISSUE THREAD

**Issue by xdk78** [shymmq/librus-client-kotlin#39]
Pls don't say _dziwne, u mnie działa_

**Comment by shymmq** [comment]
But seriously, provide the logcat then :p

**Comment by xdk78** [comment]
``` 10-09 07:38:17.493 E/InputDispatcher(3165): channel '3abed77 com.wabadaba.dziennik/com.wabadaba.dziennik.ui.MainActivity (server)' ~ Channel is unrecoverably broken and will be disposed! ```

**Comment by xdk78** [comment]
```
I/art: Rejecting re-init on previously-failed class java.lang.Class<io.requery.sql.ManagedTransaction>                                                                                                        
I/art: Rejecting re-init on previously-failed class java.lang.Class<io.requery.sql.ManagedTransaction>                                                                                                        
I/MainActivity: 1 users logged in                                                                                                                                                                             
I/System.out: Checking token valid                                                                                                                                                                            
I/System.out: Token refresh failed                                                                                                                                                                            
W/System.err: io.reactivex.exceptions.OnErrorNotImplementedException: Request to URL https://api.librus.pl/2.0/Me failed with message: Device Offline                                                         
W/System.err:     at io.reactivex.internal.observers.CallbackCompletableObserver.accept(CallbackCompletableObserver.java:46)                                                                                  
W/System.err:     at io.reactivex.internal.observers.CallbackCompletableObserver.accept(CallbackCompletableObserver.java:25)                                                                                  
W/System.err:     at io.reactivex.internal.observers.CallbackCompletableObserver.onError(CallbackCompletableObserver.java:63)                                                                                 
W/System.err:     at io.reactivex.internal.operators.completable.CompletableResumeNext$ResumeNext$OnErrorObserver.onError(CompletableResumeNext.java:95)                                                      
W/System.err:     at io.reactivex.internal.disposables.EmptyDisposable.error(EmptyDisposable.java:73)                                                                                                         
W/System.err:     at io.reactivex.internal.operators.completable.CompletableError.subscribeActual(CompletableError.java:29)                                                                                   
W/System.err:     at io.reactivex.Completable.subscribe(Completable.java:1635)                                                                                                                                
W/System.err:     at io.reactivex.internal.operators.completable.CompletableResumeNext$ResumeNext.onError(CompletableResumeNext.java:78)                                                                      
W/System.err:     at io.reactivex.internal.operators.maybe.MaybeIgnoreElementCompletable$IgnoreMaybeObserver.onError(MaybeIgnoreElementCompletable.java:73)                                                   
W/System.err:     at io.reactivex.internal.operators.maybe.MaybeFromSingle$FromSingleObserver.onError(MaybeFromSingle.java:82)                                                                                
W/System.err:     at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.onError(SingleSubscribeOn.java:73)                                                                          
W/System.err:     at io.reactivex.internal.operators.single.SingleCreate$Emitter.tryOnError(SingleCreate.java:95)                                                                                             
W/System.err:     at io.reactivex.internal.operators.single.SingleCreate$Emitter.onError(SingleCreate.java:81)                                                                                                
W/System.err:     at com.wabadaba.dziennik.api.RxHttpClient$executeCall$1.subscribe(RxHttpClient.kt:26)                                                                                                       
W/System.err:     at io.reactivex.internal.operators.single.SingleCreate.subscribeActual(SingleCreate.java:39)                                                                                                
W/System.err:     at io.reactivex.Single.subscribe(Single.java:2700)                                                                                                                                          
W/System.err:     at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)                                                                              
W/System.err:     at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:452)                                                                                                                               
W/System.err:     at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:61)                                                                                                        
W/System.err:     at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:52)                                                                                                       
W/System.err:     at java.util.concurrent.FutureTask.run(FutureTask.java:237)                                                                                                                                 
W/System.err:     at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:154)                                                                    
W/System.err:     at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:269)                                                                           
W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)                                                                                                          
W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)                                                                                                          
W/System.err:     at java.lang.Thread.run(Thread.java:833)                                                                                                                                                    
W/System.err: Caused by: com.wabadaba.dziennik.api.HttpException$DeviceOffline: Request to URL https://api.librus.pl/2.0/Me failed with message: Device Offline                                               
W/System.err: 	... 13 more                                                                                                                                                                                    
I/Bugsnag: Saved unsent error to disk (/data/user/0/com.wabadaba.dziennik/cache/bugsnag-errors/1508170094024.json)                                                                                            
I/Bugsnag: Sending 2 saved error(s) to Bugsnag                                                                                                                                                                
W/Bugsnag: Could not send previously saved error(s) to Bugsnag, will try again later                                                                                                                          
           com.bugsnag.android.ErrorReportApiClient$NetworkException: Network error when posting to https://notify.bugsnag.com                                                                                
               at com.bugsnag.android.DefaultHttpClient.postReport(DefaultHttpClient.java:24)                                                                                                                 
               at com.bugsnag.android.ErrorStore$1.run(ErrorStore.java:66)                                                                                                                                    
               at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)                                                                                                             
               at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)                                                                                                             
               at java.lang.Thread.run(Thread.java:833)                                                                                                                                                       
            Caused by: java.lang.RuntimeException: No network connection available                                                                                                                            
               at com.bugsnag.android.DefaultHttpClient.postReport(DefaultHttpClient.java:24)                                                                                                                 
               at com.bugsnag.android.ErrorStore$1.run(ErrorStore.java:66)                                                                                                                                    
               at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)                                                                                                             
               at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)                                                                                                             
               at java.lang.Thread.run(Thread.java:833)                                                                                                                                                       
W/Bugsnag: Could not send previously saved error(s) to Bugsnag, will try again later                                                                                                                          
           com.bugsnag.android.ErrorReportApiClient$NetworkException: Network error when posting to https://notify.bugsnag.com                                                                                
               at com.bugsnag.android.DefaultHttpClient.postReport(DefaultHttpClient.java:24)                                                                                                                 
               at com.bugsnag.android.ErrorStore$1.run(ErrorStore.java:66)                                                                                                                                    
               at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)                                                                                                             
               at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)                                                                                                             
               at java.lang.Thread.run(Thread.java:833)                                                                                                                                                       
            Caused by: java.lang.RuntimeException: No network connection available                                                                                                                            
               at com.bugsnag.android.DefaultHttpClient.postReport(DefaultHttpClient.java:24)                                                                                                                 
               at com.bugsnag.android.ErrorStore$1.run(ErrorStore.java:66)                                                                                                                                    
               at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)                                                                                                             
               at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)                                                                                                             
               at java.lang.Thread.run(Thread.java:833)                                                                                                                                                       
I/Process: Sending signal. PID: 10007 SIG: 9                                                                                                                                                                  
```

**Comment by xdk78** [comment]
Resolved in 0.4.2
